app/src/main/AndroidManifest.xml

```diff
@@ -14,7 +14,7 @@
         android:label="${appName}"
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
-        tools:replace="allowBackup">
+        tools:replace="allowBackup, android:label">
```

Comment:
Why?

Reply:
Because of following build error by TimeLineView lib.
`AndroidManifest.xml:14:9-35 Error:
	Attribute application@label value=(eventyay organizer) from AndroidManifest.xml:14:9-35
	is also present at [com.github.vipulasri:timelineview:1.0.6] AndroidManifest.xml:13:9-41 value=(@string/app_name).
	Suggestion: add 'tools:replace="android:label"' to <application> element at AndroidManifest.xml:10:5-63:19 to override.`

Reply:
Alright